src/contracts/interfaces/IVoteWeigher.sol

```diff
@@ -34,17 +34,20 @@ interface IVoteWeigher {
      * @notice This function computes the total weight of the @param operator in the quorum @param quorumNumber.
      * @dev returns zero in the case that `quorumNumber` is greater than or equal to `NUMBER_OF_QUORUMS`
      */
-    function weightOfOperator(address operator, uint8 quorumNumber) external returns (uint96);
+    function weightOfOperator(uint8 quorumNumber, address operator) external returns (uint96);
 
     /// @notice Number of quorums that are being used by the middleware.
-    function quorumCount() external view returns (uint8);
+    function quorumCount() external view returns (uint16);
 
     /**
      * @notice This defines the earnings split between different quorums. Mapping is quorumNumber => BIPS which the quorum earns, out of the total earnings.
      * @dev The sum of all entries, i.e. sum(quorumBips[0] through quorumBips[NUMBER_OF_QUORUMS - 1]) should *always* be 10,000!
      */
     function quorumBips(uint8 quorumNumber) external view returns (uint256);
 
+    /// @notice Returns the strategy and weight multiplier for the `index`'th strategy in the quorum `quorumNumber`
+    function strategyAndWeightingMultiplierForQuorumByIndex(uint8 quorumNumber, uint256 index) external view returns (StrategyAndWeightingMultiplier memory);
```

Comment:
we should add a getter for the length of these if we don't have it already

Reply:
already got one